speakeasy/winenv/api/kernelmode/ntoskrnl.py

```diff
@@ -83,6 +83,12 @@ def KeServiceDescriptorTable(self, ptr=0):
 
         return desc_tbl
 
+    @impdata('KdDebuggerEnabled')
+    def KdDebuggerEnabled(self, ptr=0):
+        if not ptr:
+            return self.mem_alloc(8, base=None, tag='emu.struct.KdDebuggerEnabled')
```

Comment:
This variable is not used by `NtQuerySystemInformation` so it's possible to detect the sandbox.